main.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+)
+
+type task struct {
+	ID       string `json:"id"`
+	Name     string `json:"name"`
+	Complete bool   `json:"complete"`
+}
+
+var tasks = []task{
+	{ID: "1", Name: "Wake up", Complete: false},
+	{ID: "2", Name: "Go to work", Complete: false},
+	{ID: "3", Name: "Make dinner", Complete: false},
+}
+
+func main() {
+	router := gin.Default()
+	router.GET("/health", getHealth)
+	router.GET("/tasks", getTasks)
+
+	router.Run("localhost:8080")
+}
+
+func getHealth(c *gin.Context) {
+	c.IndentedJSON(http.StatusOK, "OK")
```

Comment:
This is a bit nitpicky, especially for a health endpoint, but it's uncommon for a json endpoint to just return a string instead of a full json object. In addition to meeting other developers expectations, a full object would allow expanding to include other information like the cause of a non-OK status